src/read.rs

```diff
@@ -46,34 +47,24 @@ impl<R: Read> ReadRng<R> {
 
 impl<R: Read> Rng for ReadRng<R> {
     fn next_u32(&mut self) -> u32 {
-        // This is designed for speed: reading a LE integer on a LE
-        // platform just involves blitting the bytes into the memory
-        // of the u32, similarly for BE on BE; avoiding byteswapping.
```

Comment:
This is probably a breaking change on big-endian? I don't think this will be a problem for anyone, but maybe good to mention.

Reply:
Yes, but only because it wasn't portable before. I doubt anyone was depending on the old non-portable behaviour.